clients/callback_client.go

```diff
@@ -48,6 +49,22 @@ func (c CallbackClient) DoWithRetries(r *http.Request) error {
 	return nil
 }
 
+func (c CallbackClient) SendRecordingEvent(event *RecordingEvent) {
+	eventJson, err := json.Marshal(event)
+	if err != nil {
+		log.Printf("SendRecordingStarted json marshal %v", err)
+		return
+	}
+	req, err := http.NewRequest(http.MethodPost, config.RecordingCallback, bytes.NewReader(eventJson))
```

Comment:
If this is run in background and errors ignored, WDYT of at least adding a couple of retries here so we don't miss recordings out of ephemeral errors?

Reply:
Good point. This is served by following `c.DoWithRetries(req)` mechanism available for all callbacks to studio.